postman-generator/src/test/scala/generator/PostmanItemBuilderSpec.scala

```diff
@@ -0,0 +1,224 @@
+package generator
+
+import examples.ExampleJson
+import io.apibuilder.spec.v0.models._
+import models.postman._
+import org.scalatest.{Assertion, Matchers, WordSpec}
+import play.api.libs.json.{JsString, JsValue, Json}
+
+class PostmanItemBuilderSpec extends WordSpec with Matchers {
```

Comment:
Does `PostmanItemBuilder` can fail / throw ? 
I just wonder about negative test.

Reply:
AFAIR - it can not throw